posts/jumpstarting-your-data-engineering-career:-a-beginner's-guide-to-the-data-stack-part-one-fivetran.md

```diff
@@ -40,12 +40,12 @@ Pros:
 Cons:
 - Limited transformation capabilities
     - Primarily designed for ingestion, so its transformation capabilities are limited
```

Comment:
Might be worth adding a little context here, as Fivetran has essentially deprecated SQL transformations and push people to leverage dbt now.

https://fivetran.com/docs/transformations/dbt

Reply:
Thanks for the help with wording this nicely!